basketball_reference_web_scraper/parsers/box_scores/play_by_play.py

```diff
@@ -0,0 +1,38 @@
+import re
+from lxml import html
+
+from basketball_reference_web_scraper.data import Location
+
+
+TIME_REGEX = "([0-9]+):([0-9]+)\.([0-9]+)"
+SCORE_REGEX = "([0-9]+)-([0-9]+)"
+
+
+def parse_play_by_plays(page):
+    tree = html.fromstring(page)
+    table = tree.xpath('//table[@id="pbp"]')
+    quarter = 0
+    result = []
+    for row in table[0][1:]:
+        if row[0].get("colspan") == "6":
+            quarter += 1
+        elif row[1].get("colspan") != "5" and row[0].get("aria-label") != "Time":
+            result.append(parse_play_by_play(row, quarter))
+    return result
+
+
+def parse_play_by_play(row, quarter):
+    time = re.search(TIME_REGEX, row[0].text_content().strip())
+    score = re.search(SCORE_REGEX, row[3].text_content().strip())
+    location = Location.HOME if row[1].text_content().strip() == "" else Location.AWAY
+    return {
+        "q": quarter,
```

Comment:
Somehow indicating the home and away teams might be really useful to API users.

Reply:
I've added the teams abbreviation to the output file name (because I don't want to add it to each line in the output)

Reply:
That's an interesting way of indicating the home and away teams - unfortunately only solves the case for end-users that are outputting to files (vs. using in-memory solution).

What about slightly changing the returned `dict` so it outputs something like

```python
{
  home_team: Team.NEW_JERSEY_NETS,
  away_team: Team.TORONTO_RAPTORS,
  plays: [
    // Here add the existing array of play result dicts
  ],
}
```

Only downside is for the CSV solution, the output would be slightly different where either it's _not_ included or it's added to every line.

Thoughts?

Reply:
Yea I've missed the point :) 
I've added the home and away teams as you said
https://github.com/Yotamho/basketball_reference_web_scraper/blob/17fc2315452fd3961f3d61811fe8a7c46960b500/basketball_reference_web_scraper/parsers/box_scores/play_by_play.py#L26-L30